docs/csharp/misc/cs0101.md

```diff
@@ -32,3 +32,25 @@ namespace MyNamespace
    }  
 }  
 ```
+
+A CS0101 is also generated when your class name clashes with your namespace name.  This can happen when expanding with helper classes for the base class where you attempt to keep the name space route the same.  In the below example, the UTF8 class should clearly be a subsidiary of the String class, but attempting to force it into the same name space by declaring said name space as Utilities.String will cause a CS0101 error:
```

Comment:
"namespace" should be one word:

```suggestion
A CS0101 is also generated when your class name clashes with your namespace name.  This can happen when expanding with helper classes for the base class where you attempt to keep the namespace route the same.  In the below example, the UTF8 class should clearly be a subsidiary of the String class, but attempting to force it into the same name space by declaring said namespace as Utilities.String will cause a CS0101 error:
```